admin/TwoWeeksReady.Admin/Components/KitItemDisplay.razor

```diff
@@ -55,7 +55,7 @@
     public async Task Save()
     {
         IsEditMode = false;
-        await repository.SaveBaseKitItem(Item);
+        @* await repository.SaveBaseKitItem(Item); *@
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, the API only has methods for the BaseKit and not the Base kit items so the method was removed from the repository. I was leaving additional updates needed for the item component for issue #65